pkg/virt-config/config-map.go

```diff
@@ -209,16 +210,18 @@ func defaultClusterConfig() *v1.KubeVirtConfiguration {
 }
 
 type ClusterConfig struct {
-	configMapInformer                cache.SharedIndexInformer
-	crdInformer                      cache.SharedIndexInformer
-	kubeVirtInformer                 cache.SharedIndexInformer
-	namespace                        string
-	lock                             *sync.Mutex
-	lastValidConfig                  *v1.KubeVirtConfiguration
-	defaultConfig                    *v1.KubeVirtConfiguration
-	lastInvalidConfigResourceVersion string
-	lastValidConfigResourceVersion   string
-	configModifiedCallback           ConfigModifiedFn
+	configMapInformer                       cache.SharedIndexInformer
+	crdInformer                             cache.SharedIndexInformer
+	kubeVirtInformer                        cache.SharedIndexInformer
+	namespace                               string
+	lock                                    *sync.Mutex
+	lastValidConfig                         *v1.KubeVirtConfiguration
+	defaultConfig                           *v1.KubeVirtConfiguration
+	lastInvalidConfigResourceVersion        string
+	lastValidConfigResourceVersion          string
+	lastInvalidHostDevConfigResourceVersion string
+	lastValidHostDevConfigResourceVersion   string
```

Comment:
are these values used anywhere? or is this left over from a previous change?

Reply:
thanks! just a leftover, removed.